Classes/Hooks/RealUrlAutoConfiguration.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types=1);
+namespace JWeiland\ServiceBw2\Hooks;
+
+/*
+ * This file is part of the service_bw2 project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+use DmitryDulepov\Realurl\Configuration\AutomaticConfigurator;
+use JWeiland\ServiceBw2\RealUrl\TitleMapping;
+
+/**
+ * Class RealUrlAutoConfiguration
+ *
+ * @package JWeiland\ServiceBw2\Hooks
+ */
+class RealUrlAutoConfiguration
+{
+    /**
+     * Generates additional RealURL configuration and merges it with provided configuration
+     *
+     * @param array $parameters
+     * @param AutomaticConfigurator $parentObject
+     *
+     * @return array Updated configuration
+     */
+    public function addConfig(array $parameters, AutomaticConfigurator $parentObject): array
+    {
+        return array_merge_recursive($parameters['config'], [
```

Comment:
array_merge_recursive is eval. If you have different string in both arrays they will be merged into into an array. No override happen. Please use ArrayUtility of Core. Don't know the method name. Something with array_merge_recursive_overrule